### PR TITLE
Initial support for X-EconomicAppIdentifier

### PR DESCRIPTION
**Pull request by prognostikos**
So here's a somewhat hacky solution.

I couldn't easily see how to test if request headers are set when mocking requests, but I have run this on a production batch job that uses `connect_with_credentials` and it does work for me.

### CODE REVIEW

File: lib/economic/endpoint.rb [hunk 1]
```diff
@@ -8,6 +8,10 @@ class Economic::Endpoint
   def_delegator "client.globals", :log_level, :log_level=
   def_delegator "client.globals", :log, :log=
 
+  def initialize(app_identifier = nil)
```
Comment (koppen) [inline lib/economic/endpoint.rb]: Can we add some documentation about what `app_identifier` is meant to be? Has E-conomic written something somewhere we can copy/paste and/or link to?

Comment (prognostikos) [inline lib/economic/endpoint.rb]: Will do. My preference would be to add the link to the blog post in the commit message and add a note to the README - sound good?

Comment (koppen) [inline lib/economic/endpoint.rb]: A note in the README sounds good. I'd prefer some rdoc-style documentation directly in the code above the method definition if possible. See for example `#call` below. This way the docs will appear on for example http://www.rubydoc.info/gems/rconomic/Economic/Endpoint

**Comment by prognostikos** [comment]
@koppen OK, I think it's pretty well documented now.

**Comment by prognostikos** [comment]
Er, well, `Session` is. Will add a comment for `Endpoint`

**Comment by koppen** [comment]
That's great, thanks!

**Pull request merged**